docker-compose/README.md

```
@@ -10,6 +10,59 @@ This is the docker-compose setup to run eSignet Signup service with mock identit
 4. Git bash
 5. Postman
 
+## Bring up the complete signup setup for a demo
+
+The [docker-compose.yml](docker-compose.yml) file brings up the complete signup setup — `signup-service` and `signup-ui` along with all the dependent services. With this setup, the signup service need not be run from the IDE.
+
+### Step 1: Generate the signup OIDC keystore
+
+1. Open a Git bash terminal in the [postman-collection](../postman-collection) folder and run:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)